docs/APK_USER_DOCUMENTATION.md

```diff
@@ -0,0 +1,41 @@
+<!--
+
+    Sonatype Nexus (TM) Open Source Version
+    Copyright (c) 2019-present Sonatype, Inc.
+    All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+
+    This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+    which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+
+    Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+    of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+    Eclipse Foundation. All other trademarks are the property of their respective owners.
+
+-->
+## APK Repositories
+
+### Introduction
+
+[APK](https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management) is the built-in package manager for Alpine Linux, a security-oriented, lightweight Linux distribution based on musl libc and Busybox.
+
+### Proxying APK repository
+
+You can create a proxy repository in Nexus Repository Manager that will cache packages from a remote APK repository, like
+http://dl-cdn.alpinelinux.org/alpine/v3.10/main/. Then, you can make `Alpine` use your Nexus Repository Proxy 
+instead of the remote repository by editing `/etc/apk/repositories`. More information can be found [here](https://wiki.alpinelinux.org/wiki/Enable_Community_Repository). You may need to first install nano/vim/etc for this to work (i.e. `apk add nano`). 
```

Comment:
nano/vim/etc looks like we have such editor "etc".  but at the same time I can put any editor name instead of "etc" @joedragons  what do u think about this construction?

Reply:
I agree with you @aornatovskyy .  Perhaps "install an editor, such as nano, vim, etc.," however, I'm not confident on my suggestion since I'm not sure what we're having work.  Editing /etc/apk/repositories?
If so, I don't personally feel we need to tell them they may need something to edit a file.

Reply:
@aornatovskyy @joedragons Thank you both! I absolutely see the confusion, and yes @joedragons, the work requested of the user is to have them edit the file located at `/etc/apk/repositories`. I'll change for clarification.

Reply:
in: https://github.com/sonatype-nexus-community/nexus-repository-apk/pull/1/commits/09471e269484ac8a27a67f73695f1a82de8bc47c